content/sdk/00.index.md

```diff
@@ -1,21 +1,147 @@
 ---
-title: zkSync Era SDKs
-description:
+title: "Home"
+menuTitle: "Introduction"
+description: "zkSync Era SDKs empower developers to seamlessly interact with the zkSync Era network."
+navigation: true
+tags: "zksync, sdks, zkSync Era SDK, ethereum, era network, javascript, go, swift, python, java, rust"
 ---
 
-zkSync Era SDKs empower developers to seamlessly interact with the zkSync Era network,
-a high-performance scaling solution for Ethereum.
+## What is zkSync Era SDKs?
+
+zkSync Era SDKs help developers create decentralized applications (dApps) for the zkSync Era network. zkSync Era is a
+Layer 2 solution for Ethereum. It uses ZK rollup technology to provide scalable and low-cost transactions, keeping the
+security and trust of the Ethereum blockchain.
+
+<div align="center">
+    <img width="50%" src="/images/sdk/zksync-era-sdks.png" alt="zkSync Era SDKs"/>
+</div>
+
+## Benefits of zkSync Era SDKs
 
 These SDKs, available in various programming languages, streamline the process of building decentralized applications
 (dApps) that leverage zkSync Era's benefits, including:
 
-- **Scalability**: Massively reduced transaction fees and faster confirmation times compared to the Ethereum.
+- **Scalability**: Massively reduced transaction fees and faster confirmation times compared to Ethereum.
 - **Security**: Inherits security from the underlying Ethereum blockchain.
 - **Enhanced User Experience**: Enables smoother and more cost-effective dApp interactions for users.
 - **Paymaster Support**: Enables flexible transaction fee payment strategies:
   - Third-party services can cover fees for users, fostering wider adoption.
   - Gasless transactions become possible, improving user experience.
 - **Account Abstraction**:
   - Enhances security and privacy by separating user accounts from keys.
-  - Improves developer experience by simplifying account management and enabling features
-    like multi-signature wallets and on-chain governance.
+  - Improves developer experience by simplifying account management and enabling features like multi-signature wallets
+    and on-chain governance.
+
+## Integration
+
+To integrate third-party SDKs with zkSync Era, follow these key steps to ensure your decentralized application (dApp) can
+use zkSync Era's scalable, secure, and low-cost transactions.
+
+::card-group
+::card
+---
+title: JavaScript
+icon: i-heroicons-code-bracket
+to: /sdk/js/ethers/ethers
+target: _blank
+---
+Integrate JS with zkSync Era for secure transactions.
+::
+::card
+---
+title: GO
+icon: i-heroicons-cog
+to: /sdk/go/getting-started
+target: _blank
+---
+Integrate zksync GO SDK for zkSync Era features.
+::
+::card
+---
+title: Python
+icon: i-heroicons-cog
+to: /sdk/python/getting-started
+target: _blank
```

Comment:
We should avoid making these `target: _blank` to open in a new tab. Recommended accessibility guidelines suggest opening within the same tab, especially for links that are internal.

Reply:
edited